### DISCUSSION TITLE
Issues with srsRAN gNB with COTS UEs Setup

### DISCUSSION THREAD

**Discussion by fahad-ece** [srsran/srsRAN_Project#474] [General Help]
Hi,

I have completed all the steps for srsRAN gNB with COTS UE setup running on Ubuntu 22.04.3 LTS + USRP N310 + 5G Enabled UE 

Right now there is no internet connectivity with the UE. Please let me know if I can solve it by anyway. Thank you.
 

https://github.com/srsran/srsRAN_Project/assets/158372277/89a3e908-45f0-433d-a686-42b884383370

[ N.B: I was following the instructions from here -[ https://docs.srsran.com/projects/project/en/latest/tutorials/source/cotsUE/source/index.html]

**Comment by robertfalkenberg** [comment]
This error message means that the gNB was not able to reach the core.

Please check your core is running and listening on 127.0.1.100:38412
